cypress/e2e/Class01.cy.js

```diff
@@ -0,0 +1,85 @@
+/// <reference types="Cypress" />
+
+describe('EngageSphere', ()=> {
+  beforeEach(()=> {
+    cy.visit('/')
+  })
+  
+  context('Heading', ()=> {
+
+    it('Shows the heading and the theme button', ()=> {
+      cy.contains('h1','EngageSphere')
+        .should('be.visible')
+      cy.get('[id="theme-toggle-button"]')
+        .should('be.visible')
+    })
+  })
+
+  context('Greeting', ()=>{
+
+    it('Validate default greting', ()=>{
+      const pastDate = new Date(Date.UTC(1988, 3, 23))
+      cy.clock(pastDate)
+      cy.visit('/')
+      cy.contains('Hi there! It is now Fri Apr 22 1988')
```

Comment:
O dia não devia ser 23?

Além disso, falta verificar a visibilidade do elemento.

Por fim, é uma boa prática separar pré-condições, ações e resultados esperados para facilitar a leitura das diferentes partes do teste. Veremo isso no último encontro da _masterclass_ quando eu falar sobre o padrão 3A. O mesmo vale para outros casos onde isso ocorre.

```suggestion

      cy.contains('Hi there! It is now Fri Apr 23 1988').should('be.visible')
```

Reply:
Quando verifico a existência de 23, o teste não passa por time out.
Estou separando dessa forma:
cy.contains('Hi there! It is now Fri Apr 22 1988')
        .should('be.visible')